R/cols-best_case_worst_case.R

```diff
@@ -0,0 +1,43 @@
+col_companies_id <- function() {
+  "companies_id"
+}
+
+col_ep_product <- function() {
+  "ep_product"
+}
+
+col_benchmark <- function() {
+  "benchmark"
+}
+
+col_emission_profile <- function() {
+  "emission_profile"
+}
```

Comment:
As I explained above, the most general concept behind this column is that of "risk category".